aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -255,6 +250,11 @@ public BaseHandlerStd(final HandlerConfig config) {
                 .backoffDelay(config.getBackoff())
                 .makeServiceCall(NOOP_CALL)
                 .stabilize((request, response, proxyInvocation, model, context) -> isDbInstanceStabilized(proxyInvocation, model))
+                .handleError((request, exception, proxyInvocation, resourceModel, context) -> Commons.handleException(
+                        ProgressEvent.progress(resourceModel, context),
+                        exception,
+                        UPDATE_ASSOCIATED_ROLES_ERROR_RULE_SET
```

Comment:
Is this should be `DB_INSTANCE_ROLE_FAILED_TO_STABILIZE` rule set?

Reply:
Decided to keep it as is, as NotStabilized is a terminal error